gui/wxpython/gui_core/gselect.py

```diff
@@ -2103,8 +2104,35 @@ def GetDsn(self):
                 "PostgreSQL",
                 "PostGIS Raster driver",
```

Comment:
Since this is changing code impacting both options, did you test both of them?

Reply:
Yes of course and it works as expected. PR for import PostGIS geometry data is here #2490.